WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -285,10 +289,10 @@ - (UINavigationController *)notificationsNavigationController
     self.notificationsViewController = [notificationsStoryboard instantiateInitialViewController];
     _notificationsNavigationController = [[UINavigationController alloc] initWithRootViewController:self.notificationsViewController];
     _notificationsNavigationController.navigationBar.translucent = NO;
-    self.notificationsTabBarImage = [[UIImage imageNamed:@"icon-tab-notifications"] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
+    self.notificationsTabBarImage = [UIImage imageNamed:@"icon-tab-notifications"];
     self.notificationsTabBarImageUnread = [[UIImage imageNamed:@"icon-tab-notifications-unread"] imageWithRenderingMode:UIImageRenderingModeAlwaysOriginal];
```

Comment:
I noticed that this is the only image that we keep rendering with `UIImageRenderingModeAlwaysOriginal` is this so we don't lose the color on the notification little orange ball?

Reply:
Yes. Unfortunately I'm not sure of another way around that (other than drawing the ball into the image at runtime).

Reply:
That's OK I was just curious, thanks :)